Simulator/src/main/kotlin/edu/drexel/se575/BlockchainSimulator.kt

```diff
@@ -101,8 +101,8 @@ fun main() {
         val address = ctx.pathParam("address")
         val amount = ctx.pathParam("amount")
         var usrAccount: Account? = null
-        blockChain.interpreter.accountList.filter {
-            usrAccount == it
+        usrAccount = blockChain.interpreter.accountList.filter {
+            address == it.address
         }
```

Comment:
I think you have to add [0] at the end since filter returns a list (like the other filter functions up top)

Reply:
Yeah, I caught that after it crashed horrifically on me. -_-